### DISCUSSION TITLE
Should the log file be translated?

### DISCUSSION THREAD

**Discussion by pkratoch** [rpm-software-management/dnf5#1062] [General]
The untranslated logs are better when investigating logs provided in bug reports, however, the translations might be better for system administrators.

The aim of this discussion is to gather information, how it's done in other projects and how the logs are used by community and customers.

**Comment by ppisar** [comment]
Logs are usually forgotten to be translated. However, they usually quote error messages from third-party programs and functions (like strerror()) and those are translated. To achieve consistency, I would translate the logs. A log is a recording of debugging messages. Software translates debugging messages in general.
